src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -332,8 +337,10 @@ public void setHoodAngle(double degrees) {
             degrees = hoodTestAngle;
         }
         degrees = Math.min(Math.max(degrees, MIN_HOOD_ANGLE), MAX_HOOD_ANGLE);
-        targetHoodAngle = degrees;
```

Comment:
What was the reason for switching from `targetHoodAngle` to `hoodTarget`? Using raw sensor inputs makes the `ALLOWED_HOOD_ERROR` constant less understandable.
(`getHoodAngle()` returns the hood angle in degrees.)